### ISSUE TITLE
urap ceramics: filemaker shows blank panel over UI after inserting an image (error type 1)

### ISSUE THREAD

**Issue by urapadmin** [urapadmin/kiosk#1257]
This here is for the image container in locus photos. Much smaller todo list on my side:

- [x] change add_*_to_cache method to the deferred strategy

**Comment by urapadmin** [comment]
### @test 11.6.4
- [ ] apply test scheme

**Comment by luizaogs** [comment]
Getting the menu between flashes because I assume you haven't changed the timing on flickers here, yada yada.

**Comment by luizaogs** [comment]
It is not possible to add an image through the popup here (other than through the gallery), correct? Or am I missing a spot?

**Comment by luizaogs** [comment]
Should ceramic images have only the ceramic number in the gallery? If I search for the unit and locus numbers they do pop up, but they only show the ceramic number: 

![IMG_0250](https://user-images.githubusercontent.com/67970402/141862908-4f9cedec-798c-4c60-bd5b-39e2256dba5b.PNG)

I ask because collected material images appear with all three identifiers, so I was not sure.

**Comment by luizaogs** [comment]
Same deleting bug as reported in #1255 and #1256 (deleting image through FM menu leads to deletion in the gallery). Not sure if that will be solved with your fix in 11.6.7 already.

**Comment by luizaogs** [comment]
The rest is good here.

**Comment by urapadmin** [comment]
before I release the hacky thing mentioned in #1255: Please try Filemaker Go 15.4 which just got released, first.

**Comment by lbestock** [comment]
Back to testing for Meghan and Laurel. What we are really testing for with all these are the image errors themselves. When we've made sure those aren't there, the other bugs found in following the testing scheme should be pulled out and made into their own separate issues.

**Comment by urapadmin** [comment]
### @test 11.6.17
Sorry we have to test this again because of a bug I found. There should be exactly ONE flicker and all should work, of course.

**Comment by mcornacchia25** [comment]
Is there a difference between this and su ceramics in #1258? If so, how do I test this? Is there a configuration setting I need to change? Or is it not in the same ceramics tab?

**Comment by luizaogs** [comment]
I managed to configure urap ceramics to test this -- are you surprised? Yeah, me too.

Still getting the flashing menu here (again noting it since I am not sure if you meant to get rid of it). And my question about which/how the identifiers should appear in the gallery from above remains -- if this is an issue I guess it should go in a new ticket, but since I do not know whether it is an issue I haven't opened one.

Deletion problem reported in #1296.

**Comment by luizaogs** [comment]
> Is there a difference between this and su ceramics in #1258? If so, how do I test this? Is there a configuration setting I need to change? Or is it not in the same ceramics tab?

Yes, I think you just need to change the stuff as seen here: #1207. (There might be more to it but this seems to have worked)

**Comment by mcornacchia25** [comment]
I think I got it to work. The arrow navigation of the images is really weird, pulling up the menu every time and not moving to the last image in a list, but I'm pretty sure I saw another ticket on this issue. The image containers themselves all seem to work with one flash, though it does pull up the menu for a second. Deleting with the FileMaker menu does cause the blank image issue in the gallery.
